### PR TITLE
Feature/31231 Display appropriate markup on out of time screen for try out checks

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
Render different markup based on the the practice config value

### CODE REVIEW

File: pupil-spa/src/app/out-of-time/out-of-time.component.html [hunk 1]
```diff
@@ -30,6 +30,15 @@ <h1 class="heading-xlarge aa-title-size" tabindex="0">Out of time</h1>
                     <a id='sign-out' routerLink="/sign-out" class="button aa-button-size wider aa-contrast-button">Sign out</a>
                 </p>
             </div>
+          <div class="column-full" *ngIf="familiarisationCheck">
+            <header class="page-header">
+              <h1 class="heading-xlarge aa-title-size" tabindex="0">Thank you</h1>
+            </header>
+              <p class="lede aa-lead-size" tabindex="0">
+                You can <a routerLink="">start again</a> if you would like to try again.
```
Comment (jon-shipley) [inline pupil-spa/src/app/out-of-time/out-of-time.component.html]: routerLink is empty?

Comment (GeorgeChatzigiannis) [reply pupil-spa/src/app/out-of-time/out-of-time.component.html]: @jon-shipley redirects to login page

Comment (jon-shipley) [reply pupil-spa/src/app/out-of-time/out-of-time.component.html]: Is there a way to make this explicit?

Comment (GeorgeChatzigiannis) [reply pupil-spa/src/app/out-of-time/out-of-time.component.html]: @jon-shipley yes, added

**Pull request merged**